### ISSUE TITLE
YAML config - Invalid character in numeric literal

### ISSUE THREAD

**Issue by bogd** [adnanh/webhook#548]
When trying to use webhook with a YAML config file, I am getting the following error:

```
[webhook] 2021/08/01 09:11:17 couldn't load hooks from file! invalid character ' ' in numeric literal
[webhook] 2021/08/01 09:11:17 serving hooks on http://0.0.0.0:9000/hooks/{id}

```
Steps to reproduce:
- take the YAML example from the documentation
```

- id: redeploy-webhook
  execute-command: "/var/scripts/redeploy.sh"
  command-working-directory: "/var/webhook"

```
- try to run webhook using it
```

# webhook -hooks webhook2.yaml --verbose
[webhook] 2021/08/01 09:11:17 version 2.5.0 starting
[webhook] 2021/08/01 09:11:17 setting up os signal watcher
[webhook] 2021/08/01 09:11:17 attempting to load hooks from webhook2.yaml
[webhook] 2021/08/01 09:11:17 couldn't load hooks from file! invalid character ' ' in numeric literal
[webhook] 2021/08/01 09:11:17 serving hooks on http://0.0.0.0:9000/hooks/{id}
[webhook] 2021/08/01 09:11:17 os signal watcher ready

```
I am using webhooks installed from apt packages, on Ubuntu 18.04.

Is there anything else I am missing from the documentation?

**Comment by bogd** [comment]
Appears to be an issue with v2.5.0 (the latest one available in packages for Ubuntu 18.04). After replacing the binary with v2.8.0, everything seems to work fine.

**Comment by bogd** [comment]
@moorereason - I had left the issue open because this might warrant at least a mention in the documentation (it took me a while to figure this out, because I went looking at the code and all I could see was a Yaml.Unmarshal when reading the input file). 

At least people facing a similar issue should be able to find this :)
